CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Aries Cloud Agent Python Changelog
 
+## 1.2.5rc0
+
+### July 14, 2025
+
+This patch release adds a GitHub Action to publish an LTS container image with the tag `1.2-lts` when a release is published from the ACA-Py 1.2.lts branch. This is a convenience for those who want to use the latest LTS version of ACA-Py in their deployments, and is not intended to be used as a replacement for the latest release of ACA-Py or when a specific release is required.
+
+The release includes the following PRs from the `main` branch cherry-picked into this release:
+
+- Put cred_rev_id read, increment and write in a transaction [\#3793](https://github.com/openwallet-foundation/acapy/pull/3793)
+- Remove header from http/ws responses [\#3753](https://github.com/openwallet-foundation/acapy/pull/3753)
+- 🐛 Fix v2 cred ex and pres ex webhook events to emit after db write [\#3699](https://github.com/openwallet-foundation/acapy/pull/3699)
```

Comment:
I missed putting this in the PR description but https://github.com/openwallet-foundation/acapy/pull/3692 was backported as well.